modules/javafx.base/src/main/java/com/sun/javafx/PlatformUtil.java

```diff
@@ -63,7 +64,7 @@
         embedded = bool1;
 
         @SuppressWarnings("removal")
-        String str2 = AccessController.doPrivileged((PrivilegedAction<String>) () -> System.getProperty("embedded"));
+        String str2 = AccessController.doPrivileged((PrivilegedAction<String>) () -> System.getProperty("glass.platform","").toLowerCase(Locale.ROOT));
```

Comment:
Minor: maybe add a space after the comma

Reply:
done